docker-build-and-publish/Dockerfile

```diff
@@ -0,0 +1,86 @@
+ARG PG_VERSION
```

Comment:
not positive this should be in here or in the repo with the code and provided in the workspace

Reply:
The dockerfile could be in the timescaleDB repo passed in via the workspace. That way you only need to provide the path to the dockerfile in the action